ls1intum/Artemis
# `Plagiarism checks`: Fix an issue with plagiarism checks for Java programming exercises when running Artemis within a Docker image

### Checklist
#### General



### Motivation and Context
Closes #5945

### Description
JPlag seems to need JDK-specific classes to compare Java submissions. Those classes are not part of the JRE. Therefore, the comparison fails with the internal server error as described in #5945.

This change increases the size of the Docker image again by 200MB to the same size it was before #5739.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Java Programming exercise with a few submissions

1. Log in to Artemis
2. Go to the exercise instructor view > ‘Check Plagiarism’ and click on Detect Plagiarism.
3. No InternalServerError should appear.


### Review Progress






#### Code Review
#### Manual Tests
Probably optional, since the change is so small.

### Test Coverage
unchanged
